### PR TITLE
Cliquet docs

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: docs/api/resource.rst [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+##################
 
-.. _http-apis:
+.. _resource-endpoints:
 
 GET /{resource}
 ===============
 
-XXX define the resource get endpoint.
+Returns all records of the current user in this resource.
```
Comment (ametaireau) [inline docs/api/resource.rst]: I would naively use "for this resource", rather than "in".

File: cliquet/resource.py [hunk 2]
```diff
@@ -54,27 +54,48 @@ def wrapper(klass):
...
+
+    * ``id``
+    * ``last_modified``
     """
     id = colander.SchemaNode(colander.String(), missing=colander.drop)
     last_modified = TimeStamp()
 
     class Options:
+        """
+        """
```
Comment (leplatrem) [inline cliquet/resource.py]: XXX: document options

File: docs/contributing.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+::
+
+    make tests
+
+
+Definition of done
+==================
+
+* Tests pass
```
Comment (Natim) [inline docs/contributing.rst]: We could add a bullet ``* new code added comes with tests.``

File: docs/installation.rst [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+Full server
+-----------
+
+In Ubuntu/Debian based::
+
+    sudo apt-get install postgresql
+
+
+Server using Docker
```
Comment (Natim) [inline docs/installation.rst]: Maybe we should add the ``sudo apt-get install docker.io``?

File: docs/rationale.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+we follow at Mozilla. The goal is to produce APIs which are easy to consume
+for the clients and follow some well known patterns.
+
+Cliquet handles:
+
+* Records validation
+* Storage by user
+* Pagination
+* Sorting and filtering
+* Preconditions headers
```
Comment (Natim) [inline docs/rationale.rst]: Record Race Condition handling using Preconditions headers.

File: docs/rationale.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+
+Context
+=======
+
+* Cloud Services team at Mozilla
+* Reading list project story
+* Firefox Sync
+* Cloud storage
+
```
Comment (Natim) [inline docs/rationale.rst]: * Firefox OS User Data synchronization

File: docs/rationale.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+Vision
+======
+
+General:
+
+* A global protocol : Cliquet as the implementation of reference in python
```
Comment (Natim) [inline docs/rationale.rst]: Python

File: docs/rationale.rst [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+* JavaScript client: implementation of reference
```
Comment (Natim) [inline docs/rationale.rst]: Maybe we should put a list of project using cliquet (clients and server)

- Readinglist-client (Web, Android, IOS)
- Readinglist server
- Kinto

File: docs/rationale.rst [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+* Cloud Services team at Mozilla
+* Reading list project story
+* Firefox Sync
+* Cloud storage
+
+
+Vision
+======
+
+General:
```
Comment (Natim) [inline docs/rationale.rst]: Strange to have title without title level.

File: docs/index.rst [hunk 10]
```diff
@@ -1,44 +1,25 @@
...
-   model
-   authentication
-   api
-   batch
-   utilities
-   timestamps
-   versionning
-   backoff
-   errors
+.. include:: ../CHANGELOG.rst
```
Comment (Natim) [inline docs/index.rst]: We should probably create a changelog file that does the include. See #33

**Pull request merged**